client/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@identity.com/cryptid",
   "version": "0.1.1",
   "license": "MIT",
-  "main": "dist/index.js",
```

Comment:
hm can you double check if we still need that? Ideally we don't change anything outside of the cli

Reply:
Added back. I also figured out why I was having the module import error... I didn't run yarn build in the client, which seems to have fixed the issue -_-